src/cockpit/389-console/src/lib/server/accessLog.jsx

```diff
@@ -361,358 +421,353 @@ export class ServerAccessLog extends React.Component {
+                                        onChange={(str, e) => {
+                                            this.handleChange(e, "exp");
                                         }}
+                                    />
+                                </GridItem>
+                            </Grid>
+                            <Grid
+                                className="ds-margin-top"
+                                title="The server deletes the oldest archived log file when available disk space is less than this amount. (nsslapd-accesslog-logminfreediskspace)."
+                            >
+                                <GridItem className="ds-label" span={3}>
+                                    Free Disk Space (in MB)
+                                </GridItem>
+                                <GridItem span={1}>
+                                    <TextInput
+                                        value={this.state['nsslapd-accesslog-logminfreediskspace']}
+                                        type="number"
+                                        id="nsslapd-accesslog-logminfreediskspace"
+                                        aria-describedby="horizontal-form-name-helper"
+                                        name="server-accesslog-logminfreediskspace"
+                                        onChange={(str, e) => {
+                                            this.handleChange(e, "exp");
+                                        }}
+                                    />
+                                </GridItem>
+                            </Grid>
+                            <Grid
+                                className="ds-margin-top"
+                                title="Server deletes an old archived log file when it is older than the specified age. (nsslapd-accesslog-logexpirationtime)."
+                            >
+                                <GridItem className="ds-label" span={3}>
+                                    Log File is Older Than ...
+                                </GridItem>
+                                <GridItem span={1}>
+                                    <TextInput
+                                        value={this.state['nsslapd-accesslog-logexpirationtime']}
+                                        type="number"
+                                        id="nsslapd-accesslog-logexpirationtime"
+                                        aria-describedby="horizontal-form-name-helper"
+                                        name="server-accesslog-logexpirationtime"
+                                        onChange={(str, e) => {
+                                            this.handleChange(e, "exp");
+                                        }}
+                                    />
+                                </GridItem>
+                                <GridItem span={1} className="ds-left-margin">
+                                    <FormSelect
```

Comment:
The same as for the 'Day' but here - for the 'Month'

Reply:
Fixed